src/common/footer/Footer.jsx

```diff
@@ -5,14 +5,14 @@ const Footer = () => {
     <footer className="app-footer text-center">
       <hr className="separater" />
       <div className="flex justify-center items-center py-2">
-        <Link to="/tech-stacks">
+        <p>
```

Comment:
it seems thing got reverted. From my view we should be only using Components coming from react-router-dom in order to do routing which prevent the default browser behaviour. unless Link component its fine to use `href` attribute but its inconsistent and making page refresh which is not an improvement from my prospective.

Reply:
Yes i accept your point @Angryman18 . Then better to add Link tag instead of the anchor tag ?